app/node/categories/categoriesMap.json

```diff
@@ -0,0 +1,9 @@
+{
+  "search": {
+    "אינטרנט": "Internet"
```

Comment:
It would be clearer if the key / value meaning was consistent. So if in exact matches the key is the category and value is the string to match, I would do the same for search

Reply:
Right, the `search` should be also **CategoryName** to **Array of values**.

Reply:
This structure is better for readability. This is why I'm changing it in the code to be **seller to category** or **search keyword to category** because when you editing the JSON, you want to see one category and all its values, but in the code, you handle one value each time, and you want the category of it.